ui/src/shared/components/DragAndDrop.tsx

```diff
@@ -175,7 +175,7 @@ class DragAndDrop extends PureComponent<Props, State> {
 
     const reader = new FileReader()
     reader.readAsText(file)
-    reader.onload = loadEvent => {
+    reader.onload = (loadEvent: any) => {
```

Comment:
is there a more specific type than any?

Reply:
The 2.7 to 2.8 upgrade introduced the type error described by [this post](https://stackoverflow.com/questions/35789498/new-typescript-1-8-4-build-error-build-property-result-does-not-exist-on-t). We could do what the recommended answer says and define

```
interface FileReaderEventTarget extends EventTarget {
    result:string
}

interface FileReaderEvent extends Event {
    target: FileReaderEventTarget;
    getMessage():string;
}
```

Reply:
In fact that sounds reasonable to me, I was just being lazy by using `any`

Reply:
Annnd nevermind, the above types don't typecheck in our situation. Going to stick with `any` for now.